e2e/utils/test-utils.ts

```diff
@@ -206,13 +202,10 @@ export async function createWorkspace(page: Page, cdrVersionName: string = confi
  * Else choose one of the suitable workspaces randomly.
  *
  * @param page
- * @param opts (all are optional)
- *  alwaysCreate - create a new workspace, regardless of whether a suitable workspace exists
- *  workspaceName - return the workspace with this name if it can be found; default behavior otherwise
+ * @param workspaceName Returns the workspace with this name if it can be found.
+ *  Otherwise, create a new workspace with this name.
  */
-export async function findOrCreateWorkspace(page: Page, opts: {alwaysCreate?: boolean, workspaceName?: string} = {}): Promise<WorkspaceCard> {
-  const {alwaysCreate = false, workspaceName} = opts;
-
+export async function findWorkspaceCard(page: Page, workspaceName?: string): Promise<WorkspaceCard> {
```

Comment:
This function should not be renamed.  It still creates workspaces if one can't be found.

Reply:
true. I'll change it back. can you review the whole PR?